pandas/core/window.py

```diff
@@ -857,13 +857,58 @@ def median(self, **kwargs):
         return self._apply('roll_median_c', 'median', **kwargs)
 
     _shared_docs['std'] = dedent("""
-    %(name)s standard deviation
+    Calculate %(name)s standard deviation.
+
+    Normalized by N-1 by default. This can be changed using the ddof argument.
```

Comment:
Can you put single backticks around ddof (like `` `ddof` ``) (because it is a parameter name)